builtin_op_importers.cpp

```diff
@@ -1143,8 +1143,61 @@ DEFINE_BUILTIN_OP_IMPORTER(Pad) {
 
 DEFINE_BUILTIN_OP_IMPORTER(Pow) {
   ASSERT(inputs.size() == 2, ErrorCode::kINVALID_NODE);
-  return combineTensorsElementwise(
-    ctx, inputs, nvinfer1::ElementWiseOperation::kPOW);
+  if( inputs.at(0).is_tensor() && inputs.at(1).is_tensor() ) {
+    // TODO: Check "broadcast" and "axis" attributes
+    return combineTensorsElementwise(
+        ctx, inputs, nvinfer1::ElementWiseOperation::kPOW);
+  } else if( inputs.at(0).is_weights() && inputs.at(1).is_weights() ) {
+    // TODO: Add weights together (i.e., constant fold)
+    return MAKE_ERROR("Both inputs are weights",
+                      ErrorCode::kUNSUPPORTED_NODE);
+  } else {
+    // We have two inputs, one is tensor, another are weights
+    auto* tensor_ptr = (inputs.at(0).is_tensor() ?
```

Comment:
The implementation is similar to the implementation of DEFINE_BUILTIN_OP_IMPORTER(Add) , same file line 304

We eventually should factor the common code into a separate function